content/influxdb/v2.0/monitor-alert/custom-checks.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Create custom checks
+seotitle: Custom checks
+description: >
+  Create custom checks with a Flux task.
+menu:
+  influxdb_2_0:
+    parent: Monitor & alert
+weight: 201
+v2.0/tags: [alerts, checks, tasks, Flux]
+---
+
+In the UI, you can create two kinds of [checks](/influxdb/v2.0/reference/glossary/#check):
+[`Threshold check`](/influxdb/v2.0/monitor-alert/checks/create/#threshold-check) and
+[`Deadman check`](/influxdb/v2.0/monitor-alert/checks/create/#deadman-check).
+
+Using a Flux task, you can create a custom check that provides a couple advantages:
+
+- Set up custom criteria for your alert (other than `threshold` and `deadman`).
+- Share custom checks across distributed instances.
```

Comment:
I'm not sure i understand this one. you can also share standard checks this way i believe.